ros/src/force_torque_sensor_handle_sim.cpp

```diff
@@ -0,0 +1,18 @@
+#include <ati_force_torque/force_torque_sensor_handle_sim.h>
+
+ForceTorqueSensorHandleSim::ForceTorqueSensorHandleSim(ros::NodeHandle& nh, std::string sensor_name, std::string output_frame) :
+    ForceTorqueSensorSim(nh), hardware_interface::ForceTorqueSensorHandle(sensor_name, output_frame, interface_force_, interface_torque_)
+{
+    transform_frame_ = output_frame;
+}
+
+void ForceTorqueSensorHandleSim::updateFTData(const ros::TimerEvent& event)
+{
+    interface_force_[0] = threshold_filtered_force.wrench.force.x*40;
```

Comment:
Why do we need this factors?

Can this be written in configuraiton file if we really need this?